<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>298</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout_6" >
    <item row="0" column="0" >
     <layout class="QGridLayout" name="gridLayout_5" >
      <item row="0" column="0" >
       <layout class="QGridLayout" name="gridLayout_3" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Block Sizes (Arrays)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QComboBox" name="XINC" />
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QComboBox" name="YINC" />
        </item>
        <item row="0" column="4" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5" >
         <widget class="QComboBox" name="ZINC" />
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <layout class="QGridLayout" name="gridLayout_4" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_UsesSizes" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string>Normalization Sizes (Values)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>    X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QDoubleSpinBox" name="NormalSizeCX" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="value" >
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="label_11" >
            <property name="text" >
             <string>     Y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QDoubleSpinBox" name="NormalSizeCY" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="value" >
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="4" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>     Z</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5" >
           <widget class="QDoubleSpinBox" name="NormalSizeCZ" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="value" >
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1" >
     <spacer name="horizontalSpacer" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>13</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" >
     <spacer name="verticalSpacer" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
